Modules/Core/Common/src/itkBuildInformation.cxx.in

```diff
@@ -87,7 +87,7 @@ BuildInformation::GetValue(const MapKeyType & key)
   {
     return it->second.m_Value;
   }
-  return {""};
+  return {};;
```

Comment:
`;;` <== a single semi-colon should have been sufficient here.

Reply:
Addressable via a follow up? Perhaps see whether this is the only instance of `;;`?

Reply:
@dzenanz Please feel free to do so, if you want to. I'm just reviewing `;;-)`

Reply:
(Leaving it up to whoever wants it) Maybe we should create it as a "Good first issue"?